frontend/common/src/components/inputPartials/InputError/InputError.tsx

```diff
@@ -1,8 +1,17 @@
 import React from "react";
+import type { FieldError, FieldErrorsImpl, Merge } from "react-hook-form";
+
+export type InputErrorError =
+  | string
+  | FieldError
+  // This is from `react-hook-form` so ignore the any
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  | Merge<FieldError, FieldErrorsImpl<any>>
+  | undefined;
 
 export interface InputErrorProps {
   isVisible: boolean;
-  error: string;
+  error: InputErrorError;
```

Comment:
Oh man, this type name is awful.  :laughing:  Would InputErrorMessage work or is that not accurate?  InputErrorReason?  InputErrorDetails?  InputErrorContext?

Reply:
Lol yeah, agreed. What about `InputFieldError`? I struggled to find a name since it is just an error.

I wish `react-form-form` exported this type. 😞